src/rSW_VegEstab.c

```diff
@@ -93,9 +93,10 @@ void onSet_SW_VES(SEXP VES, LOG_INFO* LogInfo) {
 			SoilWatAll.VegEstab.use = TRUE;
 			for (i = 0; i < nSPPS; i++)
 				onSet_SW_VES_spp(VES, i, LogInfo); // sets `SW_VegEstab.count` incrementally
-				if (LogInfo->stopRun) {
-				    goto report; // Exit function prematurely due to error
-				}
+
+            if (LogInfo->stopRun) {

```

Comment:
Re: Silence compiler warning [-Wmisleading-indentation]

-> Since we call `onSet_SW_VES_spp(..., LogInfo)` in a loop, the indentation was actually correct - it just lacked the correct set of `{}` to enclose both the function call plus the error check